docs/v2.5/15_lootboxes.md

```diff
@@ -283,4 +283,7 @@ Click on the lootbox and you will receive the rewards!
 
 ![](v2Assets/RewardsLootBoxes.png)
 
-We hope you enjoy bringing our new feature to life! We've found it's a great way to easily offer virtual items to any and all users with a gamified feel.
\ No newline at end of file
+We hope you enjoy bringing our new feature to life! We've found it's a great way to easily offer virtual items to any and all users with a gamified feel.
+
+Small note if you are building a Unity Lootbox Demo scene
+* When building the Lootbox Sample Scene, just make it a 0th scene in the build index. If you want to build the auth sample, make SampleLogin 0th index and then build. Since we don't support (yet) running both samples (so auth and Lootboxes) at the same time, and that can cause some unexpected behavior especially on Lootbox scene.
```

Comment:
```suggestion
When building the Lootbox Sample Scene, just make it a 0th scene in the build index. If you want to build the auth sample, make SampleLogin 0th index and then build. Since we don't support (yet) running both samples (so auth and Lootboxes) at the same time, and that can cause some unexpected behavior especially on Lootbox scene.

:::
```